lib/openhab/dsl/items/item_delegate.rb

```diff
@@ -41,6 +41,8 @@ def def_item_delegator(item, *methods)
             logger.trace("Creating delegate for (#{method}) to #{item}) for #{self.class}")
             def_delegator item, method
           end
+          # Provide uniform access to the underlying oh_item
+          def_delegator item, :java_object, :oh_item
```

Comment:
Does this work for all item types with this addition, or do the other classes need to be adapted somehow?

Reply:
yes, should work for all delegated item types.